src/dmd/e2ir.d

```diff
@@ -6227,3 +6288,36 @@ elem* elAssign(elem* e1, elem* e2, Type t, type* tx)
     }
     return e;
 }
+
+/**************************************************
+ * Initialize the dual-context array with the context pointers.
+ * Params:
+ *      loc = line and file of what line to show usage for
+ *      irs = current context to get the second context from
+ *      fd = the target function
+ *      ethis2 = dual-context array
+ *      ethis = the first context
+ *      eside = where to store the assignment expressions
+ * Returns:
+ *      `ethis2` if successful, null otherwise
+ */
+elem *setEthis2(const ref Loc loc, IRState *irs, FuncDeclaration fd, elem *ethis2, elem **ethis, elem **eside)
```

Comment:
Pointers are types in D so they are left associative (not right like in C/C++). Please write 

```d
elem* setEthis2(const ref Loc loc, IRState* irs, FuncDeclaration fd, elem* ethis2, elem** ethis, elem** eside)
```

Ditto throughout.

Reply:
done